code/modules/species/species.dm

```diff
@@ -197,6 +197,7 @@
 
 	var/list/skin_overlays = list()
 
+	var/list/limb_mapping
```

Comment:
Suggest including a comment here about the expected format (or a comment saying this is internal).

Reply:
This has been split into a separate PR and a comment has been added.